commands/summary_test.go

```diff
@@ -29,8 +29,10 @@ import (
 	"github.com/percona/pmm-admin/agentlocal"
 )
 
+const defaultPMMAgentListenPort = 7777
```

Comment:
Let's add this to `main.go`. This way, its use would be obvious when configuring kingping as well as testing here.